draft-davidson-pp-architecture.md

```diff
@@ -1320,14 +1342,21 @@ relative to the setting of A.
 We make the following assumptions in these parameter choices.
 
 - Inferring the identity of a user in a 5000-strong anonymity set is
-  difficult
+  difficult.
 - After 2 weeks, all clients in a system will have rotated to the new
-  key
+  key.
+
+In terms of additional metadata, the only concrete applications of
```

Comment:
The initial Trust Token experiment actually has log2(6) metadata bits (allowing 3 different keys to be configured at a time for public metadata and 1 bit of private metadata), though stricter allowed issuers per page limits than here. For the purpose of this spec, I'd guess keeping it at 1 metadata bit as an example, rather than using the amounts from Trust Token.